### PR TITLE
update to work with typescript@4

### PR DESCRIPTION
**Pull request by patrickkettner**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,9 +37,6 @@
   "engines": {
     "node": ">=10.13.0"
   },
-  "peerDependencies": {
-    "typescript": "^2.4.1 || ^3.0.0"
-  },
```
Comment (jirutka) [inline package.json:42]: Why did you remove `peerDependencies`?

File: test/e2e.test.ts [hunk 2]
```diff
@@ -15,9 +15,9 @@ const readFile = (filename: string) => fs.readFileSync(filename, 'utf8')
 test('build fixture project', t => {
   process.chdir(fixtureDir)
 
-  exec('yarn install', { stdio: 'inherit' })
-  exec('yarn clean', { stdio: 'inherit' })
-  exec('yarn build', { stdio: 'inherit' })
+  exec('npx yarn install', { stdio: 'inherit' })
+  exec('npx yarn clean', { stdio: 'inherit' })
+  exec('npx yarn build', { stdio: 'inherit' })
```
Comment (jirutka) [inline test/e2e.test.ts]: This is nonsense, yarn is a replacement for npm and npx.

Comment (patrickkettner) [reply test/e2e.test.ts]: I didn't have yarn installed, so the tests fail. happy to change it back

File: transformer.ts [hunk 3]
```diff
@@ -72,7 +72,7 @@ function visitImportClause (node: ts.ImportDeclaration, program: ts.Program): ts
   }
 
   const importSymbol = program.getTypeChecker().getSymbolAtLocation(node.moduleSpecifier)
-  if (!importSymbol || !isOurStubModule(importSymbol.valueDeclaration.getSourceFile())) {
+  if (!importSymbol || !isOurStubModule(importSymbol.valueDeclaration!.getSourceFile())) {
```
Comment (jirutka) [inline transformer.ts]: This will fail in runtime if `valueDeclaration` is undefined for some reason.

Comment (patrickkettner) [reply transformer.ts]: Yes.
It was previously optional, but isn't any longer. Happy to make it more robust, but given that it wouldn't introduce a new error (just continue to fail on a previously existing one), wasn't sure if you would rather this or to introduce nullcheck logic.

happy either way

File: package.json (hunk 1 above)
Comment (patrickkettner) [reply package.json:42]: the current value is exclusive of v4 - is there a specific format you would like to see?

**Comment by jirutka** [comment]
I’ve updated all the dependencies, fixed compatibility with TypeScript 4+ and released v0.2.0.
